overlord/configstate/tasksets.go

```diff
@@ -22,7 +22,7 @@ package configstate
 import (
 	"fmt"
 
-	"github.com/snapcore/snapd/i18n"
+	"github.com/snapcore/snapd/i18n/dumb"
```

Comment:
Aww, it'd be much nicer if we could just do the dumb inside i18n (which is in snapd already so we can tweak it all we like).